Update NodemcuWeather.ino

kidexx opened the pull request:
Now compatible with Arduino Json version 6.

kidexx committed:
Update NodemcuWeather.ino

Now compatible with Arduino Json version 6.

KrzysztofGre commented:
What are the versions of libraries?
There is also a problem with TFT_eSPI.
It's a pity, nice design

kidexx commented:
It is not my design, I updated only the JSON part - to version 6.17.2. I used this project as a basis (without any issues with the TFI library) and modified it very strong in the meanwhile on my local computer. I managed also to replace the whole Openwaether API call through the new API https://openweathermap.org/api/one-call-api - this required though again an update of the JSON call, but that is all not in this patch included.
